README.md

```diff
@@ -1,22 +1,54 @@
 # console-game
 Easter egg console.log game in the text based RPG fashion.
 
-## Development Server
+## Making a Game
 
-Install node.js and run this to install dependencies:
+To make a game call the NewGame function, passing it a game name (string) and options object.
 
-```
-npm install
-```
-
-Then this to start the server:
+Structure of the options object:
 
 ```
-npm start
```

Comment:
Please add this part back, probably at the end of the file. Its a simple project right now, but eventually we will have a build process that will make `npm install` and `npm start` seem more relevent.